### PR TITLE
Is_fully_completed function

### PR DESCRIPTION
**Pull request by lamprini-koutsokera**
Function for checking if a graph is full completed.

### CODE REVIEW

File: networkx/classes/function.py [hunk 1]
```diff
@@ -773,3 +773,57 @@ def is_empty(G):
...
+    ...                         (6, 2, 15), (6, 3, 18), (6, 4, 12), (6, 5, 10),
+    ...                         (6, 7, 10), (7, 1, 15), (7, 2, 14), (7, 3, 10),
+    ...                         (7, 4, 11), (7, 5, 13)])
+    >>> nx.is_full_completed(G)
+    False
+    """
+
+    nodes = G.number_of_nodes()
+    edges = G.number_of_edges()
+    return edges == (nodes * nodes - nodes)
```
Comment (MridulS) [inline networkx/classes/function.py]: This is for Directed Graph, not for general graphs.

Comment (MridulS) [inline networkx/classes/function.py]: Also according to wolfram 
```
A complete graph in which each edge is bidirected is called a complete directed graph.
```
and wikipedia
```
A complete digraph is a directed graph in which every pair of distinct vertices is connected by a pair of unique edges (one in each direction).
```

Comment (MridulS) [inline networkx/classes/function.py]: So there is no such thing as a complete (uni)directed graph. It has to be a MultiDirected Graph.

**Comment by MridulS** [comment]
I guess we can have something like 
``` python
@not_implemented_for('multigraph')
def is_complete(G):
    if G.is_directed():
        H = G.to_undirected(reciprocal=True)
        nodes = H.number_of_nodes()
        edges = H.number_of_edges()
    else:
        nodes = G.number_of_nodes()
        edges = G.number_of_edges()
    return edges == nodes *( nodes - 1)/2
```

**Comment by MridulS** [comment]
For writing tests you can use `nx.complete_graph()` to generate complete graphs.

**Comment by CrazyPython** [comment]
@lamprini-koutsokera @hagberg close or what? Just reminding in case.

**Comment by CrazyPython** [comment]
I think this should be closed - as @hagberg it's simple to test a graph's completeness.

**Comment by CrazyPython** [comment]
This should be included in a recipes section.

**Pull request closed without merging**